src/Monolog/Handler/Handler.php

```diff
@@ -44,19 +44,29 @@ public function __destruct()
         }
     }
 
-    public function __sleep()
+    public function __serialize(): array
     {
         $this->close();
 
         $reflClass = new \ReflectionClass($this);
```

Comment:
all the reflection stuff is needless:
```suggestion
        return (array) $data;
```

there *could* be some optimization by removing mangled prefixes, except if any deep private prop is to be serialized. But if the size of the serialized payload doesn't matter, the cast to array is all that's needed.

Note that if a child class implements `__sleep()`, this patch will break them. Accounting for that needs more involving code. Maybe not worth it.